src/csharp/Grpc.Tools.CSharp.targets

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
```

Comment:
What is the minimum Visual Studio version for this .targets file to work? 2013, 2015, 2017?

Reply:
I specifically avoided using any MsBuild features beyond those in v4.0. So likely the build part will work even with VS2010, if anyone is antiquated enough to still use it. The XAML object model file is required for "new" style .NET SDK projects within VS; without them, proto files will not show up in Visual Studio at all. Using this file has no effect on older build, as they are simply ignoring the new collection. The AvailableItems collection is another way to tell VS to show files in the tree, used for the "old" style .csproj, and is certainly recognized by VS2012, but I am not about of VS2010.

Stand-alone SDK projects or standalone invocations of MSBuild do not recognize any of these special collections.

Reply:
I think I was wrong about VS2010--did it support .NET4.0 at all? 

Question: what *is* the minimum supported build tools version for gRPC in .NET? I am asking this after following up on https://github.com/google/protobuf/issues/3843; In case we should proceed with a build task DLL for protoc, then we need to make only sure the task can be loaded into MSBuild v4.0, v12.0, v14.0, v15.0 for net45+ and v15.0 for netstandard1.3+ (the latter on multiple platforms).

For reference, v4.0 is used by VS2010 aka VS10.0, v12.0 by VS2013 aka VS12.0, v14.0 by VS2015 aka VS14.0, v15.0 by VS2017 aka VS15.0. Microsoft certainly made their versioning easy for us so we can have even more fun! :-/

Reply:
For gRPC as a user (= user downloads gRPC nugets and uses them in a project) we are supporting VS2013+ (.NET4.5+) and mono 4.0+ (things might actually work with mono 3.X unofficially).

For gRPC as a developer, you need VS2017 or dotnet CLI core  as we are using C#6 and the "new" .csproj XML projects, but that's a different story.

Reply:
Good to know, thanks. I'll make sure these all work.